packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -11,6 +11,7 @@ import { defineConfig, devices } from '@playwright/test';
  */
 export default defineConfig({
   testDir: './<%= directory %>',
+  outputDir: '<%= offsetFromRoot %>dist/playwright/<%= projectRoot %>/test-output',
```

Comment:
Add a note that if this path changes, then `project.json` should update too so outputs are cached.

Also, maybe an edge case, but if `dist/playwright` is already used by another project then there is a conflict. Maybe `dist/.playwright/<%= projectRoot %>` or `dist/<%= projectRoot%>-playwright` would be better?

Reply:
it should be fine since each project will be isolated by it's project path. it's the same we have configured for cypress
```
<offsetFromRoot>/cypress/<projectPath>/<screenshots|videos>
```